files/image_config/interfaces/interfaces.j2

```diff
@@ -43,58 +43,61 @@ iface eth0 inet dhcp
 {% endif %}
 #
 {% endblock mgmt_interface %}
+
 {% block front_panel_interfaces %}
 # The switch front panel interfaces
+{% for port in minigraph_ports.keys()|sort %}
+auto {{ port }}
+allow-hotplug {{ port }}
+{% endfor %}
+
 {% for interface in minigraph_interfaces %}
-auto {{ interface['alias'] }}
-allow-hotplug {{ interface['alias'] }}
-iface {{ interface['alias'] }} {{ 'inet' if interface['addr'] | ipv4 else 'inet6' }} static
+iface {{ interface['attachto'] }} {{ 'inet' if interface['addr'] | ipv4 else 'inet6' }} static
     address {{ interface['addr'] }}
     netmask {{ interface['mask'] }}
-#
 {% endfor %}
-{% for vlan_interface in minigraph_vlan_interfaces|unique_name %}
```

Comment:
I feel it's OK to keep it?

---
In reply to: [108321887](https://github.com/Azure/sonic-buildimage/pull/448#discussion_r108321887) [](ancestors = 108321887)